CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+### Added
+- OE SDK Release packages can now be built on non-SGX and non-FLC machines.
+
+### Changed
+
+### Removed
```

Comment:
One additional thing: this probably doesn't need to be in the change log, I think the `Added` information is the important one. I'll leave it up to you.

Reply:
Removed this note from the PR.